packages/suite/src/support/messages.ts

```diff
@@ -1419,6 +1419,10 @@ export default defineMessages({
         defaultMessage: 'Swap',
         id: 'TR_COINMARKET_SWAP',
     },
+    TR_COINMARKET_SWAPS: {
+        defaultMessage: 'swaps',
```

Comment:
I find it quit dangerous to have it lowercase here. I am scared that in some languages somebody will uppercase it without the context. As mentioned above I would add it here with numbers/dates,...

Reply:
Updated